tests/unittests/config/test_apt_key.py

```diff
@@ -138,7 +138,9 @@ def test_apt_key_list_fail_no_keys_file(self, m_listdir, m_gpg, *args):
             "list", m_gpg
         )
 
-    def test_apt_key_list_fail_bad_key_file(self, m_gpg):
+    @mock.patch.object(cc_apt_configure.os, "listdir", return_value=())
+    @mock.patch.object(cc_apt_configure.os.path, "isfile", return_value=False)
+    def test_apt_key_list_fail_bad_key_file(self, m_listdir, m_gpg):
```

Comment:
```suggestion
    def test_apt_key_list_fail_bad_key_file(self, m_isfile, m_listdir, m_gpg):
```

Without this change, m_gpg takes the value of the mock rather than using the expected fixture.

Line 126 has the same problem.